code/game/atoms_movable.dm

```diff
@@ -575,3 +576,67 @@
 //Can it be moved by a shuttle?
 /atom/movable/proc/can_shuttle_move(var/datum/shuttle/S)
 	return 1
+
+/atom/movable/proc/Process_Spacemove(check_drift)
```

Comment:
It only checks for dense and anchored atoms. So it's only going to happen with vehicles and mechas, and should be pretty uncommon